lib/features/family/features/home_screen/presentation/pages/family_home_screen.dart

```diff
@@ -331,6 +335,21 @@ class _FamilyHomeScreenState extends State<FamilyHomeScreen> {
     }
   }
 
+  void _showSecondParentDialog(Profile profile) {
+    showDialog<void>(
```

Comment:
FunDialog also as .show() method i think to trigger the dialog.
And is there an amplitude event?

Reply:
Will add the analytics event and take a look at the .show

Reply:
Added the show but the fundialog automatically adds the analyticsevent to the button. Also before that there's the analytics event of the avatar tapping.